.env.example

```diff
@@ -1,3 +1,5 @@
 VITE_CKAN_URL=https://data.stadt-zuerich.ch
 VITE_ANALYTICS_SRC=''
 VITE_ROBOTS_DIRECTIVE=noindex
+VITE_BACKEND_URL = https://ckan-ogdzh.clients.liip.ch
```

Comment:
please also make sure to document the `.env`-changes so that we do not forget to make them for INTEG specifically